packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/templates/expandableStandaloneSearchBox.css

```diff
@@ -26,7 +26,6 @@ textarea.searchbox__input {
 }
 
 .searchbox_floating-container {
-  position: absolute;
```

Comment:
Here I removed the `position: absolute` as well despite being 'standalone' just in the event where some content or element happens to be close to the searchbox and it hides the content/element like it does for the `expandableSearchBox`

Reply:
Did you try it in the header of the community though?
You'll need to make an aura component

```
<aura:component implements="forceCommunity:searchInterface" access="global">
    <c-quantic-standalone-search-box
            engine-id="something"
            textarea>
        </c-quantic-standalone-search-box>
</aura:component>
```

And that should work to have a draggable component in the community header